### PR TITLE
fix(image) fix typo in label name

### PR DESCRIPTION
**Pull request by Tieske**
see https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys

### CODE REVIEW

File: release-kong.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ elif [ "$RESTY_IMAGE_BASE" == "alpine" ]; then
   echo "FROM ${DOCKER_REPOSITORY}:${ARCHITECTURE}-${DOCKER_TAG}" | docker build \
     --label org.opencontainers.image.version="${KONG_VERSION}" \
     --label org.opencontainers.created="${DOCKER_LABEL_CREATED}" \
```
Comment (hutchic) [inline release-kong.sh:63]:
```suggestion
    --label org.opencontainers.image.created="${DOCKER_LABEL_CREATED}" \
```

Comment (hutchic) [reply release-kong.sh:63]: Per the spec I got this wrong too .... sorry

**Review by hutchic — APPROVED** [review]
sorry I missed you created this already 👍

File: release-kong.sh [hunk 2]
```diff
@@ -50,7 +50,7 @@ elif [ "$RESTY_IMAGE_BASE" == "alpine" ]; then
   echo "FROM ${DOCKER_REPOSITORY}:${ARCHITECTURE}-${KONG_VERSION}" | docker build \
     --label org.opencontainers.image.version="${KONG_VERSION}" \
     --label org.opencontainers.created="${DOCKER_LABEL_CREATED}" \
```
Comment (Tieske) [inline release-kong.sh:52]:
```suggestion
    --label org.opencontainers.image.created="${DOCKER_LABEL_CREATED}" \
```

**Pull request merged**